### ISSUE TITLE
Swap axis

### ISSUE THREAD

**Issue by hkkkpang** [EnricoAhlers/agi2nerf#11]
Hi,

Nice work. Why do you swap axis as this line https://github.com/EnricoAhlers/agi2nerf/blob/f10c758a710e691807578bea40bbccc24bbd43c2/agi2nerf.py#L152C6-L152C6?

**Comment by prraoo** [comment]
Yes, I have the same doubt, it would be really helpful to know. Thank you

**Comment by Ben-Mack** [comment]
@EnricoAhlers I'm having the same question, do you remember the reason for this swapping order?
